performance/tasks/cumul_smoke.yaml

```diff
@@ -0,0 +1,143 @@
+{% set flavor = flavor or "m1.tiny" %}
+{% set image = image or "^cirros.*uec$" %}
+{% set availability_zone = availability_zone or "nova" %}
+{% set start_cidr = start_cidr or "10.2.0.0/24" %}
+{% set ext_network = ext_network or "public" %}
+---
+
+  KeystoneBasic.create_and_delete_role:
+    -
+      runner:
+        type: "constant"
+        times: 1
+        concurrency: 1
+
+  NovaServers.boot_and_delete_server:
+    -
+      args:
+        flavor:
+            name: "{{flavor}}"
+        image:
+            name: "{{image}}"
+        auto_assign_nic: true
+      runner:
+        type: "constant"
+        times: 1
+        concurrency: 1
+      context:
+        users:
```

Comment:
Do not create users. We should use already existing in system (to reduce duration of task)